core/tests/webdriverio_utils/SkillEditorPage.js

```diff
@@ -234,6 +241,23 @@ var SkillEditorPage = function() {
     expect(await saveOrPublishSkillButton.isEnabled()).toEqual(false);
   };
 
+  this.saveChangesToQuestion = async function(commitMessage) {
+    await general.scrollToTop();
+    await action.click('Save Question Button', saveQuestionButton);
+    await action.setValue('Commit message', commitMessageField, commitMessage);
+    await action.click('Close save modal button', closeSaveModalButton);
+    await waitFor.invisibilityOf(
+      closeSaveModalButton, 'Save modal takes too long to disappear.');
+    await waitFor.pageToFullyLoad();
+  };
+
+  this.expectQuestionInteractionIdToMatch = async function(interactionId) {
+    var text = await action.getText(
+      'Question Interaction Id',
+      questionInteractionId);
```

Comment:
Where is this defined? Don't refer to global vars from within functions.

Reply:
Are you talking about 'questionInteractionId' ? If yes, then ditto.